src/protobuf-net.Core/Meta/SchemaGenerationOptions.cs

```diff
@@ -68,6 +68,8 @@ public enum SchemaGenerationFlags
         /// Record the sub-type relationship formally in schemas
         /// </summary>
         PreserveSubType = 1 << 1,
+
+        IncludeEnumNamePrefix = 1 << 2,
```

Comment:
needs intellisense comment (/// <summary> etc) - it is meant to fail to build in release mode without this (although I haven't done a checkout to test it)